src/leveled_pclerk.erl

```diff
@@ -183,7 +186,15 @@ merge(SrcLevel, Manifest, RootPath, OptsSST) ->
             leveled_log:log("PC023",
                             [SrcLevel + 1, FCnt, AvgMem, MaxFN, MaxP, MaxMem])
     end,
-    Src = leveled_pmanifest:mergefile_selector(Manifest, SrcLevel),
+    SelectMethod =
+        case leveled_rand:uniform(100) of
+            R when R < ?GROOMING_PERC ->
```

Comment:
```suggestion
            R when R =< ?GROOMING_PERC ->
```
I guess that if you happen to set ?GROOMING_PERC to 100, you want to do this always, whereas 0 means, never. Since rand returns number between 1 and 100, a `less equal` is more accurate

Reply:
Agreed and updated